o1vm/publish-e2e-testing-cache.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+START=$(date +%s)
+# ARCH="amd64"
+DOCKER_FILE_NAME="Dockerfile-e2e-testing-cache"
+DOCKER_HUB_USER_NAME="o1labs"
+DOCKER_HUB_REPO_NAME="proof-systems"
+DOCKER_HUB_IMAGE_TAG="o1vm-e2e-testing-cache"
+DOCKER_IMAGE_FULL_NAME="${DOCKER_HUB_USER_NAME}/${DOCKER_HUB_REPO_NAME}:${DOCKER_HUB_IMAGE_TAG}"
+
+echo ""
+echo "Preparing the file system..."
+echo ""
+zip -r -q o1vm-e2e-testing-cache.zip op-program-db-for-latest-l2-block \
+  env-for-latest-l2-block.sh \
```

Comment:
Nit: could we instead use the block number in the name?
I know it is annoying to rebuild the docker image, so maybe later if one day we change the data.

Reply:
Naming artifacts with dynamic parts is not good idea since it will impact other places. You can view that shell script file and see what the block is or would you maybe like to have separate plain text file with latest block number value inside?

Reply:
I don't have a strong opinion. Let's leave it like this if you prefer :).